### ISSUE TITLE
Provide missing labels for form elements and select

### ISSUE THREAD

**Issue by glowcloud** [swagger-api/swagger-ui#10698]
Provide labels for form elements/selects., e.g.:

- parameters
- request body
- request body inputs for `application/x-www-form-urlencoded`, `multipart/form-data`
- server select
- indication that the parameters, request body, etc. are required
- top bar URL
- filter/search (visible when filter option is enabled)
